docs/troubleshooting.md

```diff
@@ -0,0 +1,199 @@
+
+### Certificate Authority
+
+Looking at the output below, we can see that the broker is running but the service-catalog is unable to communicate with it
+because `certificate signed by unknown authority`:
+
+```
+$ oc get pods
+NAME          READY     STATUS    RESTARTS   AGE
+asb-1-xzjqx   2/2       Running   0          4s
+
+$ oc describe servicebroker
+Name:           ansible-service-broker
+Namespace:
+Labels:         <none>
+Annotations:    openshift.io/generated-by=OpenShiftNewApp
+API Version:    servicecatalog.k8s.io/v1alpha1
+Kind:           ServiceBroker
+Metadata:
+  Creation Timestamp:   2017-10-05T17:21:01Z
+  Finalizers:
+    kubernetes-incubator/service-catalog
+  Generation:           1
+  Resource Version:     21
+  Self Link:            /apis/servicecatalog.k8s.io/v1alpha1/servicebrokers/ansible-service-broker
+  UID:                  8f3782b5-a9f1-11e7-ab6a-0242ac110006
+Spec:
+  Auth Info:
+    Bearer:
+      Secret Ref:
+        Kind:           Secret
+        Name:           ansibleservicebroker-client
+        Namespace:      ansible-service-broker
+  Relist Behavior:      Duration
+  Relist Duration:      15m0s
+  Relist Requests:      0
+  URL:                  https://asb.ansible-service-broker.svc:1338/ansible-service-broker/
+Status:
+  Conditions:
+    Last Transition Time:       2017-10-05T17:22:01Z
+    Message:                    Error fetching catalog. Error getting broker catalog for broker "ansible-service-broker": Get https://asb.ansible-service-broker.svc:1338/ansible-service-broker/v2/catalog: x509: certificate signed by unknown authority
+    Reason:                     ErrorFetchingCatalog
+    Status:                     False
+    Type:                       Ready
+  Operation Start Time:         2017-10-05T17:22:02Z
+  Reconciled Generation:        0
+Events:
+  FirstSeen     LastSeen        Count   From                                    SubObjectPath   Type            Reason                  Message
+  ---------     --------        -----   ----                                    -------------   --------        ------                  -------
+  6s            6s              1       service-catalog-controller-manager                      Warning         ErrorFetchingCatalog    Error getting broker catalog for broker "ansible-service-broker": Get https://asb.ansible-service-broker.svc:1338/ansible-service-broker/v2/catalog: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
+  6s            2s              9       service-catalog-controller-manager                      Warning         ErrorFetchingCatalog    Error getting broker catalog for broker "ansible-service-broker": Get https://asb.ansible-service-broker.svc:1338/ansible-service-broker/v2/catalog: x509: certificate signed by unknown authority
+```
+
+#### Resolution: Provide caBundle to service-catalog
+
+We can get the caBundle with the following command:
```

Comment:
We need to provide the service-catalog with the `caBundle` so that it can validate the certificate signing chain. We can get the `caBundle` with the following command: